static/js/clientHooks.js

```diff
@@ -155,26 +159,30 @@ exports.postToolbarInit = function (hook_name, args) {
 }
 
 exports.postAceInit = function(hook, context) {
-  var contextMenu = require('ep_mypads/static/js/contextMenu');
+  $.getScript("http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.1/jquery-ui.min.js", function(script, textStatus, jqXHR) {});
```

Comment:
Only execute the code that relies on this script within the callback, to guarantee that its already available at the time of execution.  I think it might be better if this $.getScript is called from within toolbarAlert.postAceInit if possible. (To emphasize that only the toolbaralert is dependent on the jquery library)